azuredeploy.parameters.region.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "location": {
+        "value": "<resource-location>"
```

Comment:
can we complete here with the specific location?

Reply:
No, it is going to be replaced by eastus2 or central use depending the parameter file genereated